integration/issue10244_loopback_linux_test.go

```diff
@@ -0,0 +1,96 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/containerd/containerd/v2/integration/images"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	runtime "k8s.io/cri-api/pkg/apis/runtime/v1"
+)
+
+func TestIssue10244LoopbackV2(t *testing.T) {
+	for i := 0; i < 2; i++ {
+		tBasename := fmt.Sprintf("case-%v", i != 0)
```

Comment:
Sorry, I miss this one. Could you please use specific name for this? like `use_internal_loopback=true or false` instead of using index. It's easy to follow when there is error

Reply:
Good point, thanks. Sure, I can modify this